apps/test/unit/lib/tools/jsdebugger/DebugConsoleTest.js

```diff
@@ -236,3 +237,130 @@ describe('The DebugConsole component', () => {
+    sinon.stub(experiments, 'isEnabled').returns(true);
+    expect(experiments.isEnabled('react-inspector')).to.equal(true);
+
+    root = mount(
+      <Provider store={getStore()}>
+        <DebugConsole showReactInspector={true} />
+      </Provider>
+    );
+  });
+
+  afterEach(() => {
+    experiments.isEnabled.restore();
+    restoreRedux();
+  });
+
+  const debugOutput = () => root.find('#debug-output');
+
+  describe('when input originates from code workspace console.log', () => {
+    it('a logged array prints an array with an expander icon', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: ['test'],
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('▶["test"]');
+    });
+
+    it('a logged string prints a string without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 'hello world',
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('"hello world"');
+    });
+
+    it('a logged integer or mathematical operation prints an integer without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 1 + 1,
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('2');
+    });
+
+    it('a logged object prints an object with an expandable arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: {foo: 'bar'},
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('▶Object {foo: "bar"}');
+    });
+  });
+
+  describe('when input originates from the command prompt in the debug console', () => {
+    it('the original array is prepended with >, and the interpreted array with an expander icon is prepended with < ', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          input: '["test"]'
+        })
+      );
+      getStore().dispatch(
+        actions.appendLog({
+          output: ['test']
+        })
+      );
+      expect(debugOutput().text()).to.equal('> ["test"]< ▶["test"]');
+    });
+
+    it('the original string is prepended with >, and the interpreted string is prepended with <', () => {
+      var input = 'hello world';
+      getStore().dispatch(
+        actions.appendLog({
+          input: `${input}`
+        })
+      );
+      getStore().dispatch(
+        actions.appendLog({
+          output: `${input}`
+        })
+      );
+      expect(debugOutput().text()).to.equal('> hello world< "hello world"');
+    });
+
+    it('the original integer or mathematical operation is prepended with >, and the interpreted integer or mathematical operation is prepended with <', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          input: '1 + 1'
+        })
+      );
+      getStore().dispatch(
+        actions.appendLog({
+          output: 1 + 1
```

Comment:
I just realized something: 1+1 will be evaluated to 2 before being passed to 'output' so line 344 is equivalent to 
`output: 2`
I don't think there is any way around it though.

Reply:
Yeah, I also couldn't think of any other way around it. I'll keep it as is.